AutocompleteClient/FW/Logic/Request/CIOQuizzesQuery.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CIOQuizzesQuery.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+/**
+ Struct encapsulating the necessary and additional parameters required to execute a search query.
+ */
+public struct CIOQuizzesQuery: CIORequestData {
+    /**
+     The id of the quiz
+     */
+    public let quizId: String
+
+    /**
+     A list of answers that be in the following formats:
+     * Single Select: "answer1"
+     * Multiple Select: "answer1, answer2"
+     * Open Text: "true"
+     * Cover Page: "seen"
+     
+     Please refer to "https://docs.constructor.io/rest_api/quiz/using_quizzes/" for additional details
+     */
+    public let answers: [String]?
```

Comment:
Should we make this a var instead? 

Since let is constant. In case people want to reuse the same query and just change the answers.